### PR TITLE
[13.0][ADD] purchase_v12_control_menu: New module

### PR DESCRIPTION
**Pull request by chienandalu**
This module recovers the Purchase control menu as it was in the former version for those who find it handy

cc @Tecnativa TT35990

### CODE REVIEW

File: purchase_control_menu/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`__:
+
+  * David Vidal
```
Comment (legalsylvain) [inline purchase_control_menu/readme/CONTRIBUTORS.rst]: as all the effective code comes from odoo, I think we should : 
- set it as contributors
- keep the license to LGPL-3.

Don't you think ?

File: purchase_control_menu/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module recovers the Purchase control menu as it was in the former version.
```
Comment (legalsylvain) [inline purchase_control_menu/readme/DESCRIPTION.rst]: could you make add a more descriptive text. For newcomers in V13 (and then V14...) it doesn't mean much.

**Review by legalsylvain — CHANGES_REQUESTED** [review]
a legal remark and a suggestion. Otherwise, LGTM.

Thanks !

**Comment by pedrobaeza** [comment]
The module name should be something more like `purchase_picking_control_menu` or similar, as the other is too generic.

**Comment by chienandalu** [comment]
Well, it recovers the whole menu, with the supplier invoices option, which was on base, thus the generic name...

**Comment by pedrobaeza** [comment]
purchase_v12_control_menu ?

File: purchase_control_menu/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (chienandalu) [reply purchase_control_menu/readme/CONTRIBUTORS.rst]: Ok, I changed it in the manifest and readme regenerated

File: purchase_control_menu/readme/DESCRIPTION.rst (hunk 2 above)
Comment (chienandalu) [reply purchase_control_menu/readme/DESCRIPTION.rst]: Right, I developed it a little more

**Comment by chienandalu** [comment]
Changes done

**Review by pedrobaeza — APPROVED** [review]
@legalsylvain is it OK for you?

**Review by legalsylvain — APPROVED** [review]
Well regarding the technical name, maybe it will make no sense when we will migrate it in V20 ! But i dont have a better idea, so let's move on !

**Comment by chienandalu** [comment]
The merge failed. Maybe the branch CIs need to be updated?

**Pull request merged**